core/src/main/scala/org/ensime/indexer/ClassfileIndexer.scala

```diff
@@ -126,51 +153,19 @@ final class ClassfileIndexer(file: FileObject) extends SLF4JLogging {
   private trait ReferenceInClassHunter {
     this: ClassVisitor =>
 
-    def clazz: RawClassfile
+    var clazz: RawClassfile
```

Comment:
we probably need to refactor this trait at some point, vars in traits is bad but also deprecated soon